.mergify.yml

```
@@ -1,8 +1,10 @@
 pull_request_rules:
   - name: Automatic merge on approval
     conditions:
-      - "status-success=checks/approved"
-      - "#approved-reviews-by>=2"
+      - "#approved-reviews-by>=1"
+      - check-success=triage
+      - check-success=CodeFactor
+      - check-success=Summary
```

```suggestion
      - check-success=build (stable)
      - check-success=CodeFactor
      - check-success=Rust project
```

I would like to suggest having only the required checks to be set as a requirement to be fullfilled before the mergify bot merges the PR. Because I think the other checks like triage and summary even if they fail could still be handled by the maintainers manually. While other checks are important and can't be skipped, for example code quality check, build checks, etc. because they are important. Without them the project would either won't work or the code quality would not be great, which is as you know is never good. So with that in mind, I would like to recommend this minor change :slightly_smiling_face: .